guides/source/webpacker.md

```diff
@@ -130,7 +130,7 @@ Within source files, `import` statements are resolved relative to the file doing
 
 Out of the box, Webpacker supports CSS and SCSS using the PostCSS processor.
 
-To include CSS code in your packs, first include your CSS files in your top-level pack file as though it was a JavaScript file. So if your CSS top-level manifest is in `app/javascript/styles/styles.scss`, you can import it with `import styles/styles`. This tells webpack to include your CSS file in the download. To actually load it in the page, you need to include a `<stylesheet_pack_tag "application">`, where the `application` is the same pack name that you were using. (Note, the docs still say you need to use `stylesheet_pack_tag`, but experimenting suggests that the CSS will load without it.)
+To include CSS code in your packs, first include your CSS files in your top-level pack file as though it was a JavaScript file. So if your CSS top-level manifest is in `app/javascript/styles/styles.scss`, you can import it with `import styles/styles`. This tells webpack to include your CSS file in the download. To actually load it in the page, include `<%= stylesheet_pack_tag "application" %>` in the view, where the `application` is the same pack name that you were using. (Note, the docs still say you need to use `stylesheet_pack_tag`, but experimenting suggests that the CSS will load without it.)
```

Comment:
> Note, the docs still say you need to use `stylesheet_pack_tag`, but experimenting suggests that the CSS will load without it.

Does this mean using `stylesheet_pack_tag` is unnecessary, and we should remove the sentence entirely?

Reply:
I think this is dependent on if you import CSS into your JS, or if you want webpack to process styles separately (with the `stylesheet_pack_tag`).

Please remove this sentence as it sounds like we're not sure how our own library works. In reality, it is more related to how you choose to bundle your assets. Both methods seem to work out of the box.

Reply:
I'd be fine removing it if someone from core is fine with it. That sentence wasn't part of my change as I mainly opened this to fix the erb tags.

Reply:
> I'd be fine removing it if someone from core is fine with it.

The core team doesn't gatekeep changes to the documentation. This guide is new and very subject to change.

> I'm honestly not sure how it works myself

You can `import './some-styles.css'` in your application pack and see style changes without `stylesheet_pack_tag` in a new Rails 6.1 app using Webpacker 5.2.1. This isn't the case for 6.0beta, but that's because CSS support is opt-in and a little more manual to setup. The guide will have to be revised when that is released.

>  That sentence wasn't part of my change as I mainly opened this to fix the erb tags.

Yes, please feel free to push this change in another commit. While we're here, it makes sense to me to change.

Reply:
Fair enough. I added another commit removing that extra sentence.